yogstation/code/modules/clothing/suits/nerd.dm

```diff
@@ -0,0 +1,247 @@
+#define SOUND_BEEP_1 add_queue('sound/voice/nerdsuit/beep_1.ogg',20)
+#define SOUND_BEEP_2 add_queue('sound/voice/nerdsuit/beep_2.ogg',20)
+#define SOUND_BEEP_3 add_queue('sound/voice/nerdsuit/beep_3.ogg',20)
+
+/obj/item/clothing/suit/armor/nerd
+	name = "\improper D.O.T.A. suit"
+	desc = "The Defenseless Operator Traversal Assistance suit is a highly experimental NanoTrasen designed protective full body harness designed to prolong the lifespan (and thus productivity) of an employee via surplus medical technology found in the abandoned part of maintenance no one seems to want to talk about. Unfortunately the research department couldn't design a helmet before the third quarter so this is definitely not spaceproof. One size fits most."
+	mob_overlay_icon = 'yogstation/icons/mob/clothing/suit/suit.dmi'
+	icon = 'yogstation/icons/obj/clothing/suits.dmi'
+	icon_state = "nerd"
+	item_state = "nerd"
+	blood_overlay_type = "suit"
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	armor = list(MELEE = 25, BULLET = 25, LASER = 25, ENERGY = 25, BOMB = 0, BIO = 100, RAD = 100, FIRE = 50, ACID = 50)
+	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/gun/energy/kinetic_accelerator, /obj/item/tank/internals/ipc_coolant, /obj/item/crowbar)
+
+	var/list/funny_signals = list(
+		COMSIG_MOB_SAY = .proc/handle_speech,
+		COMSIG_MOB_DEATH = .proc/handle_death,
+		COMSIG_LIVING_IGNITED = .proc/handle_ignite,
+		COMSIG_LIVING_ELECTROCUTE_ACT = .proc/handle_shock,
+		COMSIG_CARBON_GAIN_WOUND = .proc/handle_wound_add,
+		COMSIG_MOB_APPLY_DAMAGE = .proc/handle_damage
+	)
+
+	var/static/list/wound_to_sound = list(
+		/datum/wound/blunt/severe = 'sound/voice/nerdsuit/minor_fracture.ogg',
+		/datum/wound/blunt/critical = 'sound/voice/nerdsuit/major_fracture.ogg',
+		/datum/wound/loss = 'sound/voice/nerdsuit/major_lacerations.ogg',
+		/datum/wound/slash/severe = 'sound/voice/nerdsuit/minor_lacerations.ogg',
+		/datum/wound/slash/critical = 'sound/voice/nerdsuit/major_lacerations.ogg',
+	)
+
+	var/list/sound_queue = list()
+
+	var/damage_notify_next = 0
```

Comment:
I have just learned of the [cooldown defines](https://github.com/yogstation13/Yogstation/blob/master/code/__DEFINES/cooldowns.dm#L59-L73), you should use those instead

Reply:
done